docs/system/wind_turbine.md

```diff
@@ -1,7 +1,147 @@
+
+#### Permanent Magnet Excitation
+
+Exitation with permanent magnets reduces the complexity and therefor maintance. For this reason, PM generators are used mostly on offshore wind farms. There is also no energy needed for exitation so they have slightly better efficieny. Considering the large amount of rare earths needed for permanent magnets though, this effect is negligible [6].
+
+The axial flux or axial gap generators are a special type of synchronous PM generators not widely found in larger applications. It can be however found frequently for small scale DIY turbines. Their flat structure and easy-to-wind coils makes them very suited for small workshops. They provide a fairly good efficiency and common materials such as wood, steel and epoxy can be used. Since the rotor becomes wider with increasing pole count, rotational inertia and centifugal forces are increased as well, though higer pole count mostly means lower rpm needed.
+
+## Electronics
+
+Depending on the generator used and to what load (machine, grid, battery, ...) the sytem will be connected, different electronic components are necessary. Since this OER is aimed at makers and for off-grid (or local tiny-grid) use cases, high power AC/DC-DC/AC setups, frequency synchronizing elements for direct coupled wind turbines or storage units like ppumped hydroelectric energy storage are not discussed.
+Most common standalone systems consist of a source, a sink and a storage unit.
+
+As a storage unit, mostly [batteries](battery) are used. While it is possible to connect a turbine directly to some types of batteries using a rectifier and a charge controller like [Hugh Piggott](#projects) suggested, it has some serious drawbacks.
+
+<figure>
+    <center>
+        <fig-caption src="system/wind_direct_battery.svg" caption="Simplified layout for direct connection to the battery after Piggott's design" style="width:100%"  num="4" />
+    </center>
+</figure>
+
+Whenever there is spike in voltage or the battery is full and no load is connected, current will be diverted to the dump load and dissipated. If the voltage of the turbine is much higher than the battery's, the power can't be used to actually load the battery. The same happens when the turbine voltage is below charging voltage. To mitigate these flaws, a rectifier and a [DC/DC-Converter](../development/dcdc_converter) can be used to either step-down or step-up the voltage needed to efficiently load the battery. This is mostly done inside a [charge controller](charge_controller) who can take several parameters into account like optimal load current at given generator voltage. Modern batteries like [Lithium-Ion](battery#lithium-ion) require a proper [Battery Management System](bms) to work safely. 
+
+<figure>
+    <center>
+        <fig-caption src="system/wind_controlled_battery.svg" caption="Simplified layout for controlled connection to the battery" style="width:100%"  num="5" />
```

Comment:
Here we still need the rectifier because the charge controller requires a DC voltage at the input.

And is it a "dump load" or a "load dump"? To me, dump load sounds more correct. A load dump could also be a pile of loads ;)

Reply:
Used the name from wikipedia: https://en.wikipedia.org/wiki/Load_dump

Reply:
Ah, but that's a different thing. That's a severe abuse scenario that can happen in vehicles.
Here they also talk about dump loads: https://www.altestore.com/store/charge-controllers/dump-loads-dump-load-controllers-c439/